bin/setup.sh

```diff
@@ -67,14 +67,14 @@ mk_setup_conf() {
 		System hostname = ${machine}
 		DNS domain name = obsd-lab.genua.de
 		Password for root = ${rootpw:-toor}
-		Public ssh key for root = ${sshkey}
+		Public ssh key for root = ${sshkey:-}
```

Comment:
I think the whole question should be optional, as its done for the userpw variable below.
The Question should not be answered with an empty string. Or does it work this way?